src/NuGet.Services.BasicSearch/ApplicationInsights.config

```diff
@@ -5,6 +5,12 @@
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.AdaptiveSamplingTelemetryProcessor, Microsoft.AI.ServerTelemetryChannel">
       <MaxTelemetryItemsPerSecond>5</MaxTelemetryItemsPerSecond>
     </Add>
+    <Add Type="NuGet.Services.Logging.TelemetryResponseCodeProcessor, NuGet.Services.Logging">
```

Comment:
This file resets every time you update AI nuget package version. Can we move the changes to code?